src/main/java/org/broadinstitute/hellbender/dev/tools/walkers/bqsr/DF_BaseRecalibrator.java

```diff
@@ -297,4 +410,24 @@ private static GenomicsOptions makeDirectPipelineOptions(String secretsFilePath)
     }
 
 
+    public static GenomicsOptions makeRemotePipelineOptions(String secretsFile, String project, String stagingLocation) throws Exception {
```

Comment:
This method now appears to be unused (only called from commented-out code). Could you delete it?